common/api-endpoints.ts

```diff
@@ -25,6 +25,7 @@ export interface SearchProjectsParams {
 export const SearchProjects = (params?: SearchProjectsParams) => {
   const cleanQuery: Omit<SearchProjectsParams, 'organizationId'> & {
```

Comment:
```suggestion
  const cleanQuery: Omit<SearchProjectsParams, 'organizationId' |  'categoryId'> & {
```

Falta adicionar essa tipagem aqui para o cleanQuery ter o tipo correto. O tipo `Omit` remove propriedades de uma interface. Nesse caso, nos estamos removendo o `organizationId` e `categoryId`